armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -269,6 +264,15 @@ echo "BITCOIN_NETWORK=testnet" > "${SYSCONFIG_PATH}/BITCOIN_NETWORK"
 
 
 # TOR --------------------------------------------------------------------------
+curl --retry 5 https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc | gpg --import
+gpg --export A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89 | apt-key add -
+if ! grep -q "deb.torproject.org" /etc/apt/sources.list; then 
+  echo "deb https://deb.torproject.org/torproject.org stretch main" >> /etc/apt/sources.list
+fi
+
+apt update
+apt -y install tor
```

Comment:
(Optional) Maybe also add `--no-install-recommends`?

Reply:
Good hint, but I'd need to check the whole build again for that. LMTM for now and I'll add / test this later.